pkg_list.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python
+
+import os
+import sys
+
+
+#path to dependent packages
+dependent_pkgs_path = '/srv/repos/dependencies/rhel7/dependencies'
```

Comment:
constant settings will break in the future if future  changes move the path. you should look at where you can get this path programmatically and also update it with the changes from the user

Reply:
I agree. Not sure as to why this was incorporated into this pull request. Ansible playbook logic does not reference pkg_list.py. If needed I will resubmit excluding this update or once merged, we can remove from master.